apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -307,7 +308,10 @@ export const ModerationNotification = (props) => {
           <p>{commentText}</p>
         </li>
         {/* FIXME add if once report added to serializer */}
-        <AiReportExplanation AiReport={translated.dummyText} />
+        <AiReportExplanation
```

Comment:
decided to leave the fixme there until after testing so we don't have to fake AI reports on dev